Tools/WebKitTestRunner/InjectedBundle/atspi/AccessibilityUIElementAtspi.cpp

```diff
@@ -300,7 +300,7 @@ static String attributesOfElement(AccessibilityUIElement& element)
 
     String description = element.description()->string();
     if (!description.isEmpty())
-        builder.append(description.utf8().data(), '\n');
+        builder.append(description.utf8().span(), '\n');
```

Comment:
How about just `description` instead of `description.utf8().span()`? The existing code seems to convert to UTF-8 then back as Latin-1, for no good reason.